pkg/common/oras/authprovider/azure/const.go

```diff
@@ -25,9 +25,11 @@ const (
 	dockerTokenLoginUsernameGUID               = "00000000-0000-0000-0000-000000000000"
 	AADResource                                = "https://containerregistry.azure.net/.default"
 	defaultACRExpiryDuration     time.Duration = 3 * time.Hour
-	defaultACREndpoint                         = "*.azurecr.io"
 )
 
-var logOpt = logger.Option{
-	ComponentType: logger.AuthProvider,
-}
+var (
+	logOpt = logger.Option{
+		ComponentType: logger.AuthProvider,
+	}
+	defaultACREndpoints = []string{"*.azurecr.io", "*.azurecr.us", "*.azurecr.cn"}
```

Comment:
we'll need to document the default list of supported domains in the follow up doc update :)